ush/config.sh.RRFS_CONUS_13km_ens

```diff
@@ -115,28 +116,27 @@ PREEXISTING_DIR_METHOD="upgrade" # "rename"
 INITIAL_CYCLEDEF="${DATE_FIRST_CYCL}1800 ${DATE_LAST_CYCL}2300 24:00:00"
 BOUNDARY_CYCLEDEF="${DATE_FIRST_CYCL}1800 ${DATE_LAST_CYCL}0000 06:00:00"
 PROD_CYCLEDEF="00 00,18-23 07 02 2022 *"
+PROD_CYCLEDEF="00 01,02,04,05,07,08,10,11,13,14,16,17,19,20,22,23 ${CYCLEDAY} ${CYCLEMONTH} 2022 *"
```

Comment:
Is there any way to simplify this? It is a very long list.

Reply:
No way to simplify if we do long forecast every three hours.